static/js/flow/emailTypes.js

```diff
@@ -19,6 +19,7 @@ export type EmailState = {
   sendError:                EmailSendError,
   fetchStatus?:             ?string,
   supportsAutomaticEmails?: boolean,
+  filters:                  ?Array<any>,
```

Comment:
would it be doable to make a `Filter` type for flow, so we could say `?Array<Filter>` here instead of `?Array<any>`?